packages/shared/src/components/profile/devcard/DevCardStatsSection.tsx

```diff
@@ -17,12 +17,14 @@ export function DevCardStatsSection({
   iconClassName,
 }: StatsSectionProps): ReactElement {
   return (
-    <span className="flex flex-row items-center gap-1">
+    <span className="flex flex-col">
       <strong>
         <h2 className="text-white typo-title3">{largeNumberFormat(amount)}</h2>
       </strong>
-      <Icon size={IconSize.XSmall} secondary className={iconClassName} />
-      <span className="text-raw-salt-90 typo-caption2">{label}</span>
+      <span className="flex items-center text-raw-salt-90 typo-caption2">
```

Comment:
This should be raw salt color always?

Reply:
I'm guessing still yes since it's background doesn't change based on the theme.